src/main/java/com/matchbook/sdk/ClientConfig.java

```diff
@@ -0,0 +1,69 @@
+package com.matchbook.sdk;
+
+public class ClientConfig {
+
+    private String url;
+    private String loginUrl;
+    private char[] username;
+    private char[] password;
+
+    private ClientConfig() {
+    }
+
+    public static class Builder {
+        private String url;
+        private String loginUrl;
+        private char[] username;
+        private char[] password;
+
+        public Builder(char[] username, char[] password) {
+            this.username = username;
+            this.password = password;
+        }
+
+        public Builder username(char[] user) {
```

Comment:
I don't think building methods for password and username are necessary if the credentials are provided in the builder constructor.

Reply:
yeah, good catch